### ISSUE TITLE
[Console] ExceptionListener formatting message error

### ISSUE THREAD

**Issue by grachevko** [symfony/symfony#22449]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.x-DEV

While running non existing command.

```
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function getName() on null in /usr/local/app/vendor/symfony/symfony/src/Symfony/Component/Console/EventListener/ExceptionListener.php:70
Stack trace:
#0 /usr/local/app/vendor/symfony/symfony/src/Symfony/Component/Console/EventListener/ExceptionListener.php(42): Symfony\Component\Console\EventListener\ExceptionListener::getInputString(Object(Symfony\Component\Console\Event\ConsoleErrorEvent))
#1 /usr/local/app/var/cache/test/srcTestProjectContainer.php(1661): Symfony\Component\Console\EventListener\ExceptionListener->onConsoleError(Object(Symfony\Component\Console\Event\ConsoleErrorEvent))
#2 [internal function]: srcTestProjectContainer->{closure}(Object(Symfony\Component\Console\Event\ConsoleErrorEvent), 'console.error', Object(Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher))
#3 /usr/local/app/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(174): call_user_func(Object(Closure), Ob in /usr/local/app/vendor/symfony/symfony/src/Symfony/Component/Console/EventListener/ExceptionListener.php on line 70
```

**Comment by ro0NL** [comment]
Probably related to #22322

**Comment by chalasr** [comment]
Indeed, fixed by #22322

**Comment by chalasr** [comment]
@fabpot #22322 is not merged yet
